app/assets/javascripts/trimEmail.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Created by elisahilprecht on 13/04/15.
+ */
+(function(){
+  document.getElementById('submitButton').onclick = function(e){
+    var emailInputField = document.getElementById('email');
+    emailInputField.value =  emailInputField.value.replace(/ /g,'');
+  };
+})();
```

Comment:
As we do the rest of the validation on the backend side, why not just call `str.trim` on the string on the Scala side?